src/components/organisms/EventModal/EventModal.tsx

```diff
@@ -72,7 +72,7 @@ export const EventModal: React.FC<EventModalProps> = ({
       <div className="EventModal__content">
         <h4 className="EventModal__title">{event.name}</h4>
         <span className="EventModal__subtitle">
-          by {event.host} in{" "}
+          {event.host && "by"} {event.host} in{" "}
```

Comment:
This doesn't look right? Presumably intended to be a ternary or similar?

Reply:
I'll remove all these — just checked and indeed `event.host` is a required parameter